content/rs/administering/database-operations/slave-ha.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: High Availability for Slave Shards
+description: 
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+When Replication is enabled and master shard fails, the slave shard is automatically promoted to a master shard 
+to maintain data availability. This creates a single point of failure until a new slave shard is manually created.
+
+To automatically avoid this single point of failure, you can configure the cluster 
+to automatically migrate the slave shard to another available node. In practice, 
+slave migration creates a new slave shard and replicates the data from the master 
+shard to the new slave shard. For example:
+
+1. Node:2 has a master shard and node:3 has the corresponding the slave shard.
+1. Node:2 fails and triggers a failover.
+1. The slave shard on node:3 is promoted to master.
+1. If slave HA is enabled, a new slave shard is created on an available node that does not hold the master shard.
+    All of the constraints of shard migration apply, such as rack-awareness.
+1. The data from the master shard is replicated to the new slave shard.
+
+## Configuring High Availability for Slave Shards
+
+You can enable slave HA either for:
+
+* Cluster - All databases in the cluster use slave HA
+* Database - Only the specific database uses slave HA
+
+By default, slave HA is set to disabled at the cluster level. You can enable it 
```

Comment:
We should make it clear that if the slave HA is disabled in the cluster level, the slave HA will be disabled for each database in the cluster, even when setting slave HA enabled for a specific database

Reply:
@Paz-Yanover See https://github.com/RedisLabs/redislabs-docs/pull/108/commits/44bcb4cbb4ad6a287b7d221fdc10721b6eb0eec7

Reply:
Approved. Well done :)